src/Coordinator/Hgs2Hpc.php

```diff
@@ -2,100 +2,24 @@
 
 namespace HelioviewerEventInterface\Coordinator;
 
-use \Socket;
 use \Exception;
 
 class Hgs2Hpc {
-    private Socket $socket;
-    private bool $connected = false;
-    public function __construct() {
-        $this->connectToPythonServer();
-    }
-
-    function __destruct() {
-        $this->close();
-    }
-
-    public function close() {
-        if ($this->connected) {
-            $this->write("quit");
-            socket_read($this->socket, 2);
-            socket_close($this->socket);
-            $this->connected = false;
-        }
-    }
-
-    /**
-     * Opens a socket connection to the python server which will perform the coordinate transforms
-     */
-    private function connectToPythonServer() {
-        $this->socket = socket_create(AF_UNIX, SOCK_STREAM, 0);
-        if (socket_connect($this->socket, "/tmp/hgs2hpc.sock")) {
-            $this->connected = true;
-            // Don't let socket hang while reading. Processing a coordinate should definitely not take more than 1 second.
-            // Keep in mind this timeout was arbitrarily chosen as an upper limit, not by profiling actual compute time.
-            socket_set_option($this->socket, SOL_SOCKET, SO_RCVTIMEO, ["sec" => 1, "usec" => 0]);
-        } else {
-            error_log("Failed to connect to the python Hgs2Hpc server. Check that it's running and that permissions allow the user running PHP to read and write to it.");
-        }
-    }
-
     /**
      * Converts latitude and longitude coordinates at the given time to HelioProjective coordinates
      */
     public function convert(float $latitude, float $longitude, string $date) {
         try {
-            if ($this->connected) {
-                $msg = "$latitude $longitude $date";
-                $this->write($msg);
-                $result = $this->read();
-                return $this->parseHpcResponse($result);
-            } else {
-                // If the socket couldn't connect, log it and return something.
-                // Returning 0's is going to make it very obvious that something is wrong... for better or worse.
-                // Returning anything actually is going to make it very obvious that something is wrong.
-                return array("x" => 0, "y" => 0);
-            }
+            $result = file_get_contents(HV_COORDINATOR_URL . "/hgs2hpc?lat=" . $latitude . "&lon=" . $longitude . "&obstime=" . $date);
```

Comment:
might be better to use guzzle for future testing, 
not a blocker

Reply:
teach me your ways, I do want to do this.

Reply:
Ok  :+1: let me know whenever you want this,